src/components/Main.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { ReactComponent as ExpandDown } from './Icons/ExpandDown.svg';
+import { ReactComponent as Plane } from './Icons/Plane.svg';
+import { ReactComponent as ArrowRight } from './Icons/ArrowRight.svg';
+
+import cn from 'classnames'
+
+const Main = () => {
+  return (
+    <article className="main">
+      <section className="tab-bar">
+        <span className="tab-bar__item">Fly With Us To</span>
+        <section className="side-cards">
+          {['01', '02', '03'].map((content) => (
+            <div key={content} className={cn('side-cards__item', {'-active': content === '01'})}>
+              <span>{content}</span>
+            </div>
+          ))}
+        </section>
+      </section>
+
+      <h2 className="main__header">Bangkok</h2>
+      <p className="main__description">A new door to paradise for just 15000</p>
+
+      <section className="main__details-container">
+        <p className="main__details">Book now till 25 Oct 2019</p>
+        <p className="main__details">Travel from 25 Oct 2019 to 25 Dec 2019</p>
+      </section>
```

Comment:
I'd rather change this:
`main__details-container` => `main__details`
`main__details` => `main__details-item`

To keep it consistent with the rest of the project

Reply:
Or even create a separate class for that: `main-details`, `main-details__item`